rinex/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[build-system]
+requires = ["maturin>=0.14"]
+build-backend = "maturin"
+
+[project]
+name = "rinex"
+requires-python = ">=3.8"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Apache Software License"
+    "License :: OSI Approved :: MIT License",
```

Comment:
the CI complains about a few missing "," here ;)

Reply:
Ah yes - added two now

Reply:
thank you